geomesa-cassandra/geomesa-cassandra-datastore/src/test/scala/org/locationtech/geomesa/cassandra/data/CassandraDataStoreTest.scala

```diff
@@ -237,7 +238,8 @@ object CassandraDataStoreTest {
       System.setProperty("cassandra.storagedir", storagedir.getPath)
 
       EmbeddedCassandraServerHelper.startEmbeddedCassandra("cassandra-config.yaml", 1200000L)
-      val cluster = new Cluster.Builder().addContactPoints(host).withPort(port).build()
+      // following line creates a cluster with read timeout of 40 minutes. If this is hit, something else is the issue.
+      val cluster = new Cluster.Builder().addContactPoints(host).withPort(port).withSocketOptions(new SocketOptions().setReadTimeoutMillis(2400000)).build().init()
```

Comment:
Yeah, that seems reasonable.  Note that this is just the unit test...

It might be worth editing the Travis CI config here: https://github.com/locationtech/geomesa/blob/master/pom.xml#L144 to have the value necessary for the Travis build...